storagenode/storagenodedb/storagenodedbtest/multidbsnapshot.go

```diff
@@ -0,0 +1,107 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package storagenodedbtest
```

Comment:
Keep the testdata in `testdata`. The `storagenodedbtest` is intended to be imported by other packages for running tests against storage node database.

Reply:
The reason `storagenodedbtest` is a separate pacakge, is because you cannot have "test-only" exported things in a package, so it was moved to another package.

Reply:
Ok, I'll move them into `testdata` and just call it `package testdata`.

Reply:
Done